specification/cognitiveservices/data-plane/LUIS/Programmatic/readme.md

```diff
@@ -70,7 +70,7 @@ csharp:
   sync-methods: None
   license-header: MICROSOFT_MIT_NO_VERSION
   azure-arm: false
-  namespace: Microsoft.Azure.CognitiveServices.Language.LUIS.Programmatic
+  namespace: Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring
```

Comment:
Seems the folder has to be renamed as well?

Reply:
+1 on this.